src/SAWScript/ASTUtil.hs

```diff
@@ -0,0 +1,37 @@
+module SAWScript.ASTUtil (
```

Comment:
A minor quibble: do you think it's worth renaming this module `SAWScript.AST.Util`? That would provide some amount of consistency with existing `*.Util` modules (e.g., `SAWScript.AutoMatch.Util` and `SAWScript.Prover.Util`).

I don't feel strongly about this, so feel free to ignore this suggestion.